### PR TITLE
initial command mode quickstart

### PR DESCRIPTION
**Pull request by maxandersen**
Adds command mode quickstart.

No tests and .s2i as couldn't find what would be reasonable to add there :)

### CODE REVIEW

File: getting-started-command-mode/pom.xml [hunk 1]
```diff
@@ -0,0 +1,123 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project>
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.acme</groupId>
+    <artifactId>getting-started-command-mode</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+    <properties>
+        <quarkus-plugin.version>999-SNAPSHOT</quarkus-plugin.version>
```
Comment (sunix) [inline getting-started-command-mode/pom.xml:10]: it has to be 1.4.0.Final or 1.4.1.Final, right? not 999-SNAPSHOT

Comment (maxandersen) [reply getting-started-command-mode/pom.xml:10]: not in development branch.

See checklist when opening a pr:
- [ ] uses the `999-SNAPSHOT` version of Quarkus

if you are asking in general - then yes, this feature was first released in 1.4.0.Final.

**Comment by maxandersen** [comment]
@cescoffier can we get these in to update docs do release blog with ref to this repo ? 

the build errors are totally unrelated to this PR afaics.

File: getting-started-command-mode/src/main/docker/Dockerfile.native [hunk 2]
```diff
@@ -0,0 +1,22 @@
+####
```
Comment (cescoffier) [inline getting-started-command-mode/src/main/docker/Dockerfile.native:1]: Do we really need dockerfiles for command line?

File: getting-started-command-mode/src/main/resources/application.properties [hunk 3]
```diff
@@ -0,0 +1,2 @@
+# Quarkus Configuration file
```
Comment (cescoffier) [inline getting-started-command-mode/src/main/resources/application.properties:1]: Can be removed.

Comment (maxandersen) [reply getting-started-command-mode/src/main/resources/application.properties:1]: this is what is there in other quickstarts and makes it so users can just open the file and tweak without having to remember where it should be put....so I kept it. Objection?

File: getting-started-command-mode/src/main/docker/Dockerfile.native (hunk 2 above)
Comment (maxandersen) [reply getting-started-command-mode/src/main/docker/Dockerfile.native:1]: if you want to build a native binary for your linux container when running on windows or osx then you do.

**Comment by maxandersen** [comment]
> I would prefer to have at least one test if possible. But I'm not sure we can?

I can't really think of any good way of doing it without needing something else than junit

File: getting-started-command-mode/pom.xml [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+        <quarkus.platform.artifact-id>quarkus-bom</quarkus.platform.artifact-id>
+        <quarkus.platform.group-id>io.quarkus</quarkus.platform.group-id>
+        <quarkus.platform.version>1.4.0.Final</quarkus.platform.version>
```
Comment (sunix) [inline getting-started-command-mode/pom.xml:13]: So should be 999-SNAPSHOT here?

Comment (maxandersen) [reply getting-started-command-mode/pom.xml]: yes, good catch. fixed!

File: getting-started-command-mode/src/main/docker/Dockerfile.native (hunk 2 above)
Comment (cescoffier) [reply getting-started-command-mode/src/main/docker/Dockerfile.native:1]: right! Make sense... How to work around the lack of cross-compilation! Docker!

File: getting-started-command-mode/src/main/resources/application.properties (hunk 3 above)
Comment (cescoffier) [reply getting-started-command-mode/src/main/resources/application.properties:1]: No, it's fine. Yes, the other quickstarts still have them.

**Comment by cescoffier** [comment]
All good for me, merging!

**Pull request merged**